ui/components/application/View.tsx

```diff
@@ -0,0 +1,359 @@
+            (n) => n.priority === 200
+          );
+
+        return (
+          <Accordion
+            open
+            id={`applicationEvent-${i}`}
+            key={applicationEvent.id}
+            heading={applicationEvent.name || ""}
+            theme="thin"
+          >
+            <TwoColumnContainer>
+              <FormSubHeading>
+                {t("application:preview.subHeading.applicationInfo")}
+              </FormSubHeading>
+              <StyledLabelValue
+                label={t("application:preview.applicationEvent.name")}
+                value={applicationEvent.name}
+              />
+              <StyledLabelValue
+                label={t("application:preview.applicationEvent.numPersons")}
+                value={applicationEvent.numPersons}
+              />
+              <StyledLabelValue
+                label={t("application:preview.applicationEvent.ageGroup")}
+                value={
+                  applicationEvent.ageGroupId
+                    ? `${
+                        ageGroupOptions[applicationEvent.ageGroupId].minimum
+                      } - ${
+                        ageGroupOptions[applicationEvent.ageGroupId].maximum
+                      }`
+                    : ""
+                }
+              />
+              <StyledLabelValue
+                label={t("application:preview.applicationEvent.purpose")}
+                value={
+                  applicationEvent.purposeId
+                    ? purposeOptions.find(
+                        (n) => n.value === applicationEvent.purposeId.toString()
+                      )?.label
+                    : ""
+                }
+              />
+              <StyledLabelValue
+                label={t("application:preview.applicationEvent.begin")}
+                value={applicationEvent.begin || ""}
+              />
+              <StyledLabelValue
+                label={t("application:preview.applicationEvent.end")}
+                value={applicationEvent.end || ""}
+              />
+              <StyledLabelValue
+                label={t("application:preview.applicationEvent.minDuration")}
+                value={formatDuration(applicationEvent.minDuration as string)}
+              />
+              <StyledLabelValue
+                label={t("application:preview.applicationEvent.maxDuration")}
+                value={formatDuration(applicationEvent.maxDuration as string)}
+              />
+              <StyledLabelValue
+                label={t("application:preview.applicationEvent.eventsPerWeek")}
+                value={applicationEvent.eventsPerWeek}
+              />
+              <div />
+            </TwoColumnContainer>
+
+            <FormSubHeading>
+              {t("application:Page1.spacesSubHeading")}
+            </FormSubHeading>
+            <UnitList>
+              {sortBy(applicationEvent.eventReservationUnits, "priority").map(
+                (reservationUnit, index) => {
```

Comment:
Why is this inline? a component seems more appropriate. It already looks like one with variable definitions and a return.